rhel8/profiles/ospp.profile

```diff
@@ -851,3 +851,5 @@ selections:
 
     ## Enable dnf-automatic Timer
     - timer_dnf-automatic_enabled
+
+    - harden_ssh_client_crypto_policy
```

Comment:
One more thing, add the selection of the rule close the other related rules, below `harden_sshd_crypto_policy` for example.